colabs/wandb-model-registry/New_Model_Logging_in_W&B.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>How does one do this? </p>

Reply:
<a href="https://colab.research.google.com/github/wandb/examples/blob/master/LOCATION/TO/REPLACE_NOTEBOOK.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>
<!--- @wandbcode{REPLACE_THIS_WITH_NAME_FOR_TELEMETRY} -->

More or less replace the heavily capitalized parts in the above. This will just capture if people are clicking to the notebook. Not sure if its actually granular